test/unit/specs/components/image-grid.spec.js

```diff
@@ -0,0 +1,82 @@
+import ImageGrid from '~/components/ImageGrid/ImageGrid'
+import { render, screen } from '@testing-library/vue'
+
+const propsData = {
+  images: undefined,
+  canLoadMore: undefined,
+  isFetching: undefined,
+  fetchingError: undefined,
+  errorMessageText: undefined,
+  isFinished: undefined,
+}
+
+test('ImageGrid loads with default values', async () => {
```

Comment:
Rather than repeating `ImageGrid` at the start of each test, could we wrap the whole suite in a `describe('ImageGrid')` block?